#!/usr/bin/env python
PACKAGE = "avoidance"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# local_planner
gen.add("min_box_x",    double_t,    0, "Minimum box extension in x direction", 10,  0, 20)
gen.add("max_box_x",    double_t,    0, "Maximum box extension in x direction", 10,  0, 20)
gen.add("min_box_y",    double_t,    0, "Minimum box extension in y direction", 10,  0, 20)
gen.add("max_box_y",    double_t,    0, "Maximum box extension in y direction", 10,  0, 20)
gen.add("min_box_z",    double_t,    0, "Minimum box extension in z direction", 2,  0, 20)
gen.add("max_box_z",    double_t,    0, "Maximum box extension in z direction", 2,  0, 20)

exit(gen.generate(PACKAGE, "avoidance", "LocalPlannerNode"))